### ISSUE TITLE
Simplified Logo

### ISSUE THREAD

**Issue by bjzilberman** [qbittorrent/qBittorrent-website#90]
Heya, been working on a simplified logo for qb, as the light outline looks rather janky and outdated in the taskbar.

I took the same gradient, at a negative 45° angle, removed the outline and placed the letters in Arial Bold, with **q** opacity being 75% white and Letter Spacing at negative 100.

![qb icon no-jank](https://i.imgur.com/XTmBTQw.png)

What do you guys think? I am up for some improvement, please shoot me your suggestions.

**Comment by nopeac** [comment]
Love it. Even if this doesn't get implemented can you provide an .ico for personal use?

**Comment by FranciscoPombal** [comment]
Looks good, but the faded "q" makes it look blurry at a glance, or is it just me?

(come to think of it, that is also the case for the existing icon...)

**Comment by AtilioA** [comment]
A yes from me. The outline doesn't look nice and I like the new boldy look

**Comment by eraychumak** [comment]
> 
> 
> Looks good, but the faded "q" makes it look blurry at a glance, or is it just me?
> 
> (come to think of it, that is also the case for the existing icon...)

Yeah I agree as well. For simplicity, I'd keep both the letters at 100% opacity.

**Comment by symonxdd** [comment]
@bjzilberman It's been a while, but did you eventually change the opacity of the q to 100% as stated by @FranciscoPombal?
Then we could at least have a modern icon for this amazing software.
